transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipelineBuilder.scala

```diff
@@ -1,9 +1,14 @@
 package org.broadinstitute.monster.hca
 
+import cats.data.Validated
 import com.spotify.scio.ScioContext
 import com.spotify.scio.coders.Coder
 import com.spotify.scio.io.ClosedTap
 import com.spotify.scio.values.SCollection
+
+import scala.util.{Failure, Success}
```

Comment:
Nitpick: `scala.*` imports are typically grouped together at the bottom of the list. I've tried to automated it but scalafmt isn't smort enough yet.

Reply:
tfw the automation does it wrong